Max/footer refactoring

maxim7137 opened the pull request:
## Фиксы
- политика конфиденциальности - ссылка
- ссылки в nav
- уменьшил количество блоков - увеличил количество элементов
- выровнял логотипы в мобилке

maxim7137 committed:
fix: политика конфиденциальности - ссылка

maxim7137 committed:
fix: меньше блоков

maxim7137 committed:
fix: логотипы в мобилке

Maestr1 commented on blocks/footer/__copyrights-paragraph/footer__copyrights-paragraph.css:
```diff
@@ -1,4 +1,4 @@
-.copyrights__paragraph {
+.footer__copyrights-paragraph {
   margin-block-start: 0;
```
еще раз обращаю внимание
Если ты хочешь убрать margin по умолчанию пиши: margin: 0;
не нужно  margin-block-start и т.д.

Maestr1 commented on blocks/footer/__links-link/footer__links-link.css:
```diff
@@ -1,10 +1,10 @@
+.footer__links-link {
   display: inline-block;
   color: inherit;
   text-decoration: none;
```
подумай над тем, что бы сделать отдельный класс для всех ссылок, что бы не переписывать одинаковые свойства по нескольку раз

maxim7137 replied:
Отдельный блок тогда надо, а я только лишнее убрал

Maestr1 commented on blocks/footer/__links-title/footer__links-title.css:
```diff
@@ -1,4 +1,4 @@
-.links__title {
+.footer__links-title {
   font-size: inherit;
   margin-block-start: 0;
```
margin: 0;

Maestr1 commented on blocks/footer/__lists-list/footer__lists-list.css:
```diff
@@ -1,31 +1,31 @@
 
-.links__list:last-of-type {
+.footer__lists-list:last-of-type {
   margin-right: calc(14.605%);
```
подумай, как избавиться от этой кучи медиазапросов и от  calc, это тут совершенно не необходимо, но сильно переусложняет код и поддержку. 
Дам наводку, используй паддинги в контейнере, а не на каждом элементе

Maestr1 commented on blocks/footer/__partners/footer__partners.css:
```diff
@@ -1,5 +1,9 @@
+  display: flex;
+  flex-basis: 145px;
+  justify-content: space-between;
+  align-items: center;
```
ты уверен, что решение на гридах лучше? Может стоит сделать на флексе?

maxim7137 replied:
да мне кажется сложноватая сетка для флексов, а гриды позволяют разместить элементы там где надо с учетом мобильной версии

Maestr1 commented on blocks/footer/__partners-caption/footer__partners-caption.css:
```diff
@@ -1,15 +1,15 @@
 
 @media (min-width: 577px) {
-  .partners__caption {
+  .footer__partners-caption {
```
лучше по умолчанию их скрыть, а на маленькой ширине показывать

maxim7137 committed:
fix: исправил что смог
- margin: 0;
- по умолчанию их скрыть, а на маленькой ширине показывать
- изменил calc
